Testnet/NonFungibleToken-Ticket/NonFungibleToken.Tests/NonFungibleTokenTests.cs

```diff
@@ -625,7 +625,7 @@ public void DelegatedTransfer_ValidTokenTransfer_MessageSender_TransfersTokenFro
         var nonFungibleToken = CreateNonFungibleToken();
         var uid = Guid.NewGuid();
 
-        string url = $"?uid={Convert.ToHexString(uid.ToByteArray().Reverse().ToArray())}&action=DelegatedTransfer&from={this.AddressToString(ownerAddress)}&to={this.AddressToString(targetAddress)}&tokenId=1";
+        string url = $"?uid={Convert.ToHexString(uid.ToByteArray().Reverse().ToArray())}&method=DelegatedTransfer&from={this.AddressToString(ownerAddress)}&to={this.AddressToString(targetAddress)}&tokenId=1";
```

Comment:
can we  use uid as string for avoid endian stuff?

Reply:
lets rather keep uid compatible with UInt128/Guid,